docs/translations/translations.json

```diff
@@ -14,9 +14,9 @@
   "loadFont": "Load the default Roboto font.",
   "installButton": "Read installation docs",
   "usage": "Usage",
-  "usageDescr": "Material-UI components work in isolation. They are self-supporting.",
+  "usageDescr": "Material-UI components work without any additional setup.",
```

Comment:
I read “work in isolation” to refer to the fact that only the components used are added to the bundle. Not sure if ther’s anything special about that with calling out, I imagine that’s the norm these days?

Reply:
> They don't need to be isolated.

:D 

I was just trying to ascribe some meaning to the sentence, in case there's a benefit that we should be highlighting. Your proposed wording is an improvement in any case.

Reply:
By isolation, I meant that they don't pollute the global scope as Bootstrap do.